.ci/cabal.project.local

```diff
@@ -1,22 +1,39 @@
 -- use latest packages from hackage
 index-state: HEAD
 
--- don't generate haddock for all our deps
+-- Generate haddock for all our dependencies. Documentation generation for our
```

Comment:
It still says `documentation: False`, I thought you could could configure packages with `package` stanza even if they aren't listed in the `packages` stanza. i.e. aren't we saying we're not generating generating haddocks for our deps?

Reply:
You're right. This is a leftover from some iterations I went through, but it isn't needed anymore. Haddock uses its own cache and uses `--enable-documentation` to force documentation for dependencies.